app/routes/task_routes.py

```diff
@@ -1 +1,99 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, jsonify
```

Comment:
```suggestion
from flask import Blueprint, request, Response
```

See my comments in `goal_routes.py` about not needing to use `jsonify` at all in our Flask application as it's designed.

Therefore, we can safely remove `jsonify` in all parts of this file.